sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/Vector3.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.MixedReality.ObjectAnchors.Models
+{
+    /// <summary> The Vector3. </summary>
+    internal partial class Vector3 : IEquatable<Vector3>
```

Comment:
Again, should this be implicitly convertible?

Reply:
Trying to make sure the wrapper class and the actual numerics class reference the same underlying data fields. It's internal and inevitable these classes get generated unfortunately, but I'd like to treat both objects like the same reference and not make getters which spit out a new object every time that has no impact on the values of the field it was derived from.

Reply:
Leveraging implicit conversion to get rid of any *.data accesses and to make that field private, accessible through the implicit conversion